maintainers/scripts/update-luarocks-packages

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p nix-prefetch-scripts luarocks-nix -i bash
+
+# You'll likely want to use
+# ``
+# nixpkgs $ maintainers/scripts/update-luarocks-packages.sh > pkgs/development/lua-modules/generated-packages.nix
```

Comment:
This comment recommends an invocation without `$1`, but there is an assertion that `$1` is non-empty.